build.rs

```diff
@@ -21,7 +21,8 @@ const MIN_RUSTC_VERSION: &'static str = "1.15.1";
 fn main() {
 	let is = rustc_version::version().unwrap();
 	let required = MIN_RUSTC_VERSION.parse().unwrap();
-	assert!(is >= required, format!("
+	assert!(is >= required,
```

Comment:
probably, the `rustfmt` is confused by the multiple lines in `format!`. Let me check if `rustfmt` has an option to format like this.